toffy/mph_inspect.py

```diff
@@ -0,0 +1,108 @@
+import os
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+
+from ark.utils import io_utils, misc_utils
+from mibi_bin_tools.bin_files import get_histograms_per_tof
+
+
+def bin_array(arr, bin_factor):
+    arr_bin = np.cumsum(arr)
+    arr_bin[bin_factor:] = arr_bin[bin_factor:] - arr_bin[:-bin_factor]
+    arr_bin = arr_bin[bin_factor::bin_factor]
+
+    return arr_bin
+
+
+def compute_mph_intensities(bin_file_dir, fov_list, mass, mass_start=-0.3, mass_stop=0.0,
+                            bin_factor=100):
+    """ Compute the median pulse height intensities for given FOVs
+
+    Args:
+        bin_file_dir (str): path to the FOV bin files
+        fov_list (list): which FOVs to include
+        mass (float): mass for the panel
+        mass_start (float): beginning of mass integration range
+        mass_stop (float): end of mass integration range
+        bin_factor (int): size of the bins for the MPH histograms, default 100
+
+    Returns:
+        pd.DataFrame containing the MPH intensity data
+
+    """
+
+    panel = pd.DataFrame([{
+        'Mass': mass,
+        'Target': None,
+        'Start': mass_start,
```

Comment:
mass_start in the notebook is 97.5, but the default here is -0.3. Which of these is the correct syntax? We should make sure to have very clear, distinct names for arguments that are offset vs a start value